ui/src/tasks/components/RunLogRow.tsx

```diff
@@ -22,15 +22,26 @@ class RunLogRow extends PureComponent<Props> {
 
     return (
       <IndexList.Row>
-        <IndexList.Cell>{this.dateTimeString(log.time)}</IndexList.Cell>
-        <IndexList.Cell>{log.message}</IndexList.Cell>
+        <IndexList.Cell>
+          <span className="run-logs--list-time">
+            {this.dateTimeString(log.time)}
+          </span>
+        </IndexList.Cell>
+        <IndexList.Cell>
+          <span className="run-logs--list-message">{log.message}</span>
+        </IndexList.Cell>
       </IndexList.Row>
     )
   }
 
   private dateTimeString(dt: Date): string {
-    const date = dt.toDateString()
-    const time = dt.toLocaleTimeString()
+    if (!dt) {
+      return ''
+    }
+
+    const newdate = new Date(dt)
+    const date = newdate.toDateString()
+    const time = newdate.toLocaleTimeString()
     const formatted = `${date} ${time}`
```

Comment:
Would it make sense to use the `DEFAULT_TIME_FORMAT` from `src/shared/constants` instead here? Could help keep datetime formatting in the app consistent.
```
import moment from 'moment'

//...

moment(dt).format(DEFAULT_TIME_FORMAT)
```

Reply:
Oh perfect! I updated it :)